src/quetz_client/cli.py

```diff
@@ -37,8 +84,15 @@ def get_client(
     # Initialize the client (do not force the env variables to be set of help on the
     # subcommands does not work without setting them)
     url = cast(str, url or os.getenv("QUETZ_SERVER_URL", ""))
-    token = cast(str, token or os.getenv("QUETZ_API_KEY", ""))
-    client = QuetzClient.from_token(url, token)
+
+    parsed_token = get_auth_token(url, token)
+
+    if isinstance(parsed_token, BearerToken):
+        client = QuetzClient.from_bearer_token(url, str(parsed_token))
+    elif isinstance(parsed_token, ApiKey):
+        client = QuetzClient.from_token(url, str(parsed_token))
+    else:
+        client = QuetzClient.from_token(url, "")
```

Comment:
I know this is the same behavior as before, but I'm wondering if this makes sense. You always need some sort of authentication token to communicate with Quetz. Should we just fail hard, when no token is provided?